examples/provider/provider.tf

```diff
@@ -7,7 +7,7 @@ terraform {
   required_providers {
     checkly = {
       source = "checkly/checkly"
-      version = "1.4.3"
+      version = "1.6.10"
```

Comment:
v1.6.9 is the current version. We'll release 1.6.10 next, which will trigger a release of the docs.